src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -8,6 +8,14 @@ import 'react-datepicker/dist/react-datepicker.css'
 
 registerLocale('enUS', enUS)
 registerLocale('ptBR', ptBR)
+registerLocale('ar', arDZ)
+registerLocale('de', de)
+registerLocale('es', es)
+registerLocale('fr', fr)
+registerLocale('it', it)
+registerLocale('ja', ja)
+registerLocale('ru', ru)
+registerLocale('zr', zhCN)
```

Comment:
I'm wondering if there is a more dynamic way of solving this. Every time we add new language support, this component will have to be updated. 

Is there a way to simply support all locales? Or, have a client pass this component the locales it supports?

Reply:
The only issue here is a couple of locales like ar and zr are not the actual names of the locale provided by date-fns/locale. Let me know if I can add a hybrid system that supports all locales and some additional locales as welll.

Reply:
Provided mechanism to dynamically load all locales.